### PR TITLE
Sitcom 761: Load and Plot VMS data

### PR DESCRIPTION
**Pull request by psferguson**
This notebook shows how to start using the VMS data, comparisons to TMA telemetry and dc accelerometers will occur on another ticket

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb [hunk 1]
```diff
@@ -0,0 +1,436 @@
+{
```
Comment (nsevilla) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: <p>muliple --&gt; multiple</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/#comment-1342394175'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

Comment (nsevilla) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #31.    def vms_data_to_frame(filename, vms_type, begin_time=None, end_time=None):
</pre><p>Suggest changing name to vms_data_to_dataframe or vms_data_to_pandas to be more explicit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/#comment-1342394179'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

**Comment by nsevilla** [comment]
Looks good to me, though currently I do not have access to summit to test it out. I will try to get that. Do you need the review to actually have run the the notebook? (I would guess so)

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb (hunk 1 above)
Comment (nsevilla) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: <p>What are the units of the acceleration data from the VMS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/#comment-1342782307'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (nsevilla) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: <p>What does the x axis show? sorry I am bit rusty on my spectrograms</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/#comment-1342782313'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (psferguson) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: ah the x axis is seconds, I have added a label to clarify

Comment (psferguson) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb:1]: added

**Review by nsevilla — APPROVED** [review]
Comments addressed, thank you.

**Pull request merged**